### DISCUSSION TITLE
To help make installation that little bit easier without needing to search online, add comments in the .env file for possible different values for each section?

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13337] [Feature Requests]
I know there's an example .env file online that is really useful, but searching, finding and reading it takes time. How about just adding possible different values as comments in the .env file for the user to go in and edit as necessary reducing the amount of time to edit the file?

e.g. moving from an sqlite db to a ~~proper~~ db system needs some 4 extra lines of config that a new user needs to go research or just guess what they are.

**Comment by rijkvanzanten** [comment]
I'm assuming you mean adding it to the `.env` that's generated through the `create-directus-project` tool?

**Reply by sxsws** [reply]
Yes, I assume so too! I'm guessing it builds it from a template and modifies the necessary strings for the db chosen during set up? If so, would it be a simple edit to the template file?

**Reply by rijkvanzanten** [reply]
It's not using a string template file exclusively. It puts together all the configured env as strings, renders it through a [liquid stub](https://github.com/directus/directus/blob/f26d862a35c587cd3fd6f15d8e9b6469d16489c7/api/src/cli/utils/create-env/env-stub.liquid), and then writes it to a file.

Just had to double check we're talking about the same env, as you could technically also manually create the .env, or insert env vars without a file in the first place, or use docker (compose) etc etc etc 🙂

**Reply by sxsws** [reply]
sorry for my newbishness!

**Reply by rijkvanzanten** [reply]
np! I had to look it up and rewrite my answer a couple times too (and I certainly can't play the newby card anymore) 😁

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
